dspace/config/spring/api/discovery.xml

```diff
@@ -61,6 +61,11 @@
                 <entry key="publicationVolume" value-ref="publicationVolume"/>
                 <entry key="periodical" value-ref="periodical"/>
                 <entry key="project" value-ref="project"/>
+                <!-- OpenAIRE4 guidelines - search for a creator that can be a Person
+                     or an OrgUnit -->
+                <entry key="openAIRECreatorConfiguration" value-ref="openAIREAuthor"/>
```

Comment:
While this configuration is being used for openAIRE, it appears to be a generic search configuration to search across both Person and OrgUnit (in one search).   As it's a generic search configuration, we likely don't need this to be prefixed with "openAIRE" -- it could be useful for other use cases.  

So, in my opinion, this could be renamed as `personOrOrganization` (as it's a combo of the existing "person" and "organization" search configurations).

Reply:
sure

Reply:
I've just renamed the bean id to `personOrOrganization`, so it can be used in other cases like adding an entity that is a contributor (person or organization).